test/contrib/rack/helpers.rb

```diff
@@ -90,9 +90,20 @@ def app
   end
 
   def setup
-    # configure our Middleware with a DummyTracer
-    @tracer = get_test_tracer()
-    Datadog.configuration[:rack][:service_name] = 'rack'
     super
+    # store the configuration and use a DummyTracer
```

Comment:
better `setup` and `teardown` to ensure options are reset to defaults after each test.